Synthesis of proposed text for issue 160 (#206)
ietf-wg-aipref/drafts: draft-ietf-aipref-vocab.md

categories of use, plus a system for associating simple
preferences to each (allow, disallow, or unknown; see {{model}}).

The categories of use in the vocabulary (see {{vocab}})
describe concrete, observable outcomes that depend on the use of assets,
they seek to avoid describing internal details of implementations
or their architecture.

This specification only seeks to ensure that preferences can be understood,

> not provide a means of ensuring that preferences are followed.

Reviewer: ```suggestion
not provide a means of ensuring that preferences are respected.
```

followed seems like the wrong word here...

Enforcement is not provided by this specification.
Preferences do not themselves create rights, obligations, or prohibitions,
either in the positive or the negative.
Other mechanisms --
technical, legal, contractual, or otherwise --
might enforce adherence to these preferences

> and thereby determine the consequences of not following

Reviewer: ```suggestion
and thereby determine the consequences of a system not respecting
```

Author: I'm OK with the passive voice here.

a stated preference.

An entity that receives usage preferences has a choice
about whether to respect the preferences it has discovered.
It makes this choice according to its understanding

> of how the asset is used,

Reviewer: ```suggestion
of how the asset is being used,
```

Author: I don't think this is an improvement.

how that usage corresponds to the usage categories
where preferences have been stated,
and the applicable legal context.

Historically, entities have used assets for a variety of reasons.
This has included a wide array of uses in applications
including safety, advertising, education, scholarship, research,
market intelligence, preservation, surveillance, interoperability, and accessibility.
Those decisions were based on a subjective analysis

> of preexisting signals about preferences

Reviewer: ```suggestion
of pre-existing signals about preferences
```

Author: My dictionary said that this was OK without the hyphen, but I've made the change.

and an understanding of how the asset would be used in that application.
The mechanisms in this document might be more direct
than previous ways of expressing preferences,
but the same fundamental challenges remain

> when deciding whether to use an asset.

Reviewer: ```suggestion
when a system decides whether to use an asset.
```
